server/models/User.js

```diff
@@ -2,18 +2,45 @@ const mongoose = require("mongoose");
 
 const userSchema = new mongoose.Schema({
   username: {
-    type: String,
-    unique: true,
+    type: String
   },
   email: {
     type: String,
     required: true,
-    unique: true,
+    unique: true
   },
   register_date: {
     type: Date,
-    default: Date.now,
+    default: Date.now
+  },
+  timezone: {
+    type: String
+  },
+  availableHours: {
+    type: {
+      times: {
+        start: String,
+        end: String
+      }
+    },
+    default: {
+      start: "08:00",
+      end: "17:00"
+    }
+  },
+  availableDays: {
+    type: [String],
+    default: ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday"]
   },
```

Comment:
I think I understand this approach, for onboarding we want to default them to Monday-Friday being available. Will Saturday and Sunday be available as well?